frontend/examples/fresh/islands/Login.tsx

```diff
@@ -0,0 +1,21 @@
+import { HANKO_API_ENDPOINT } from "../config.ts";
+
+const code = `
+  import { register } from 'https://esm.sh/@teamhanko/hanko-elements@0.2.2-alpha';
```

Comment:
That's a pretty old version. Could you update to https://esm.sh/@teamhanko/hanko-elements@0.5.5-beta ?